docs/ocean/leap_authorization.rst

```diff
@@ -81,7 +133,7 @@ commands and all their options with the CLI's ``--help`` option.
     terminal's "``Authorization code:``" prompt, similar to the representative
     shell lines shown below::
 
-        $ dwave auth login --oob
+        $ dwave setup --oob
```

Comment:
If we decide to go with `setup` instead of `auth login`, then the output below has to be updated accordingly as well.

Reply:
Updated